### PR TITLE
Create hero section

### PR DESCRIPTION
**Pull request by Al-Amin-Khan-Shakil**
### Hero Section
In this pull request, I am adding a hero section to my portfolio that contains:

- Header section
-Navigation bar 
-Banner
-Introduction

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    </header>
+
+    <div class="banner">
+
+      <div class="introduction">
+
+        <div class="title">
+          <div class="self">Hey there.</div>
+          <div class="self">I'm Alex.</div>
+          <div class="profession">I’m a software developer</div>
```
Comment (souad988) [inline index.html]: Great work done in this section 💪🥳 but we need to use hx tags for titles this is for semantic reasons as well as for better accessibility

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        <div class="info">
+          <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+          </p>
+        </div>
+        <div class="social">
+          <img src="./image/twitter.png" alt="twitter icon" class="twitter">
+          <img src="./image/Linkedin icon.png" alt="Linkedin icon" class="linkedin">
+          <img src="./image/medium.png" alt="Medium icon" class="medium">
+          <img src="./image/github.png" alt="Github icon" class="github">
```
Comment (souad988) [inline index.html]: Well done here as well your icons are correctly aligned 👌 I suggest just that you wrap each img in an anchor a tag as it is meant to be a link that redirects the user to your social media accounts also it's a good practice to use ul element for list of items for semantic reasons

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Al-Amin-Khan-Shakil , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966


## Highlights
✔No linter errors
✔Good design 💪
✔Use of correct github flows

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          <ul>
+            <li><a href="#"><img src="./image/twitter.png" alt="twitter icon" class="twitter"></a></li>
+            <li><a href="#"><img src="./image/Linkedin icon.png" alt="Linkedin icon" class="linkedin"></a></li>
+            <li><a href="#"><img src="./image/medium.png" alt="Medium icon" class="medium"></a></li>
+            <li><a href="#"><img src="./image/github.png" alt="Github icon" class="github"></a></li>
+            <li><a href="#"><img src="./image/angellist.png" alt="Angellist icon" class="angellist"></a></li>
+          </ul>
+        </div>
+
+      </div>
```
Comment (Johnadibe) [inline index.html]: > [OPTIONAL] We apologize as this wasn't pointed out by the previous code reviewer🖐. Please make sure to enclose the headline section within a `section` tag. This tag represents a self-contained section of the document and is recommended to be used. For more information on this tag, you can refer to this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section).👍

File: style.css [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+  margin-top: 15px;
+}
+
+/* social media icon */
+
+.social ul {
+  display: flex;
+  justify-content: center;
+}
+
```
Comment (Johnadibe) [inline style.css]: > [OPTIONAL] We apologize as this wasn't pointed out by the previous code reviewer🖐. Kindly ensure to include the margin-top and font size in your design to closely match the Figma design as much as possible.☺
> 
> Figma design👇:
> 
> 2Screenshot
> 
> Your design👇:
> 
> 1Screenshot
> 
>

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @Al-Amin-Khan-Shakil👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Incorporated the changes as requested✔️
- Linters are passing✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**